docs/quick-start.md

```diff
@@ -376,15 +373,175 @@ curl -i --url http://127.0.0.1:8428/api/v1/query --url-query 'query=demo_counter
+This article provides a solid introduction to key concepts such as alert rules, recording rules, alert templates, routing, silencing, and more.
+In this guide, we’ll show how to set up alerting in Kubernetes using VictoriaMetrics tools.
+
+We’ll deploy the following resources:
+- `VMAlertManager` – handles alert notifications
+- `VMAlert` – checks alert and recording rules
+- `VMRule` – defines the alert rules
+
+We’ll also use the demo app from the [Scraping](#scraping) section again. 
+It will be used both as a source of metrics and as a webhook receiver for alerts.
+
+Create a [VMAlertmanager](https://docs.victoriametrics.com/operator/resources/vmalertmanager/) manifest and apply it:
+```sh
+cat <<EOF > vmalertmanager-demo.yaml
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMAlertmanager
+metadata:
+  name: demo
+  namespace: vm
+spec:
+  configRawYaml: |
+    route:
+      receiver: 'demo-app'
+    receivers:
+    - name: 'demo-app'
+      webhook_configs:
+      - url: 'http://demo-app.default.svc:8080/alerting/webhook'
+EOF
+
+kubectl apply -f vmalertmanager-demo.yaml;
+
+# vmalertmanager.operator.victoriametrics.com/demo created
+```
+
+Create a [VMAlert](https://docs.victoriametrics.com/operator/resources/vmalert/) manifest and apply it:
+```sh
+cat <<'EOF' > vmalert-demo.yaml
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMAlert
+metadata:
+  name: demo
+  namespace: vm
+spec:
+  # Metrics source (VMStorage/VMCluster/VMSingle)
+  datasource:
+    url: "http://vmsingle-demo.vm.svc:8429"
+
+  # Where to alert state is stored
+  remoteWrite:
+    url: "http://vmsingle-demo.vm.svc:8429"
+
+  # Where the previous alert state is loaded from. Optional
+  remoteRead:
+    url: "http://vmsingle-demo.vm.svc:8429"
+
+  # Alertmanager URL for sending alerts
+  notifier:
+    url: "http://vmalertmanager-demo.vm.svc:9093"
+
+  # How often the rules are evaluated
+  evaluationInterval: "30s"
+
+  # Watch VMRule resources in all namespaces
+  selectAllByDefault: true
+EOF
+
+kubectl apply -f vmalert-demo.yaml;
+
+# vmalert.operator.victoriametrics.com/demo created
+```
+
+You can use the same method we used before to check if the resources are running.
+Run `kubectl get pod`, but use the label selectors `app.kubernetes.io/name=vmalertmanager` and `app.kubernetes.io/name=vmalert`.
+
+Now, when you have `VMAlert` and `VMAlertManager` running, you can create your first  resource to define the alerting rules.
+This script creates a [VMRule](https://docs.victoriametrics.com/operator/resources/vmrule/) resource that defines an alert called `DemoAlertFiring`. 
+The rule expression checks if the metric `demo_alert_firing` is above zero for any `demo-app` pod in the `default` namespace. 
+If the condition holds for 30 seconds, the alert fires.
```

Comment:
maybe drop 30s thing from there and config for simplicity sake?
Anyway, `for: 30s` and `evaluationInterval:30s` don't make sense.

Reply:
fixed